src/components/LogInPage/LoginPage.js

```diff
@@ -1,6 +1,5 @@
 import { Box, Container, Flex, Heading, Spacer, Text } from "@chakra-ui/react";
 import LoginForm from "components/LoginForm/LoginForm";
```

Comment:
It is better to have a space between the imports and the react component. Was this intentional?